plugin/remotecall/main.go

```diff
@@ -0,0 +1,115 @@
+//go:build full || e2e
+
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/tg123/sshpiper/libplugin"
+	"github.com/tg123/sshpiper/plugin/internal/remotecall"
+	"github.com/urfave/cli/v2"
+)
+
+func createRemoteCaller(c *cli.Context) (*remotecall.RemoteCall, error) {
+	remoteCall, err := remotecall.InitRemoteCall(
+		c.String(userClusterEndpoint),
+		c.Generic(remoteEndpoints).(*remotecall.StringMapFlag).Value,
+		c.Path(mappingKeyPath),
```

Comment:
what does this file contain ? Is it part of different PR ?
Can we add that in this same PR so it can be tested independently.

Reply:
its required to be added together